app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardScreen.kt

```diff
@@ -125,7 +123,7 @@ fun KeyboardScreen(
             Box(
                 modifier = Modifier
                     .weight(1f) // Take up available space equally
-                    .padding(keyBorderSize),
+                    .padding(20.dp),
```

Comment:
Something weird is going on here.

Reply:
woops, the key split layout has no emoji key so I missed this, just realised the backdrop doesn't work on the emoji key either so may do another pr to fix that.